apps/arweave/src/ar_config.erl

```diff
@@ -34,6 +34,16 @@ parse(Config) when is_binary(Config) ->
 parse_storage_module(IOList) ->
 	Bin = iolist_to_binary(IOList),
 	case binary:split(Bin, <<",">>, [global]) of
+		[PartitionNumberBin, PackingBin, <<"repack_in_place">>, ToPackingBin] ->
+			PartitionNumber = binary_to_integer(PartitionNumberBin),
+			true = PartitionNumber >= 0,
+			parse_storage_module(PartitionNumber, ?PARTITION_SIZE, PackingBin, ToPackingBin);
```

Comment:
This means we only support repacking full 3.6TB storage modules in place, right?

Reply:
oops, never mind. saw the bit below